docs/CONTRIBUTING.md

```
@@ -0,0 +1,239 @@
+## Project Setup
+
+### Prerequisites
+Make sure you have the following installed:
+- **Node.js** 20+
+- **pnpm** 10.13.1+
+- **Git**
+
+### Cloning and Installing
+
+# Fork the repository (on GitHub)
+git fork https://github.com/AquaStark/Aqua-Stark.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)